src/main/java/org/terasology/tutorialWorldGeneration/HouseRasterizer.java

```diff
@@ -40,20 +41,20 @@ public void initialize() {
     public void generateChunk(CoreChunk chunk, Region chunkRegion) {
         HouseFacet houseFacet = chunkRegion.getFacet(HouseFacet.class);
 
-        for (Entry<BaseVector3i, House> entry : houseFacet.getWorldEntries().entrySet()) {
+        for (Entry<Vector3ic, House> entry : houseFacet.getWorldEntries().entrySet()) {
             // there should be a house here
             // create a couple 3d regions to help iterate through the cube shape, inside and out
             Vector3i centerHousePosition = new Vector3i(entry.getKey());
             int extent = entry.getValue().getExtent();
             centerHousePosition.add(0, extent, 0);
-            Region3i walls = Region3i.createFromCenterExtents(centerHousePosition, extent);
-            Region3i inside = Region3i.createFromCenterExtents(centerHousePosition, extent - 1);
+            BlockRegion walls = BlockRegion.createFromCenterExtents(centerHousePosition, extent);
+            BlockRegion inside = BlockRegion.createFromCenterExtents(centerHousePosition, extent - 1);
```

Comment:
`BlockRegion` does not have a member function `createFromCenterExtents`
please try to compile the code before pushing
```suggestion
            BlockRegion walls = BlockRegion(centerHousePosition).expand(extent);
            BlockRegion inside = BlockRegion(centerHousePosition).expand(extent - 1);
```

Reply:
Is nt it, BlockRegion walls = new BlockRegion(centerHousePosition).expand(0,extent,0); ?